### DISCUSSION TITLE
part-ii/2018-57/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#63] [Question comments]
Part II 2018 Fluid Dynamics II: Paper 2, Section II, C 

https://questions.tripos.org/part-ii/2018-57/

**Comment by acornnroca** [comment]
<details><summary>[answer]</summary>

$$\sigma = \frac{Uk}{\sqrt{\tanh} \pm i}$$ for varicose, $$\sigma = \frac{Uk}{\sqrt{\coth} \pm i}$$ for sinuous

</details>
